includes/wikia/WikiaSQLIterator.class.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * WikiaSQLIterator
+ */
+
+class WikiaSQLIterator {
```

Comment:
That sounds like a great idea!  Updating the PR now ...

On Fri, Feb 6, 2015 at 8:41 PM, Władysław Bodzek <notifications@github.com>
wrote:

> In includes/wikia/WikiaSQLIterator.class.php
> <https://github.com/Wikia/app/pull/6208#discussion_r24286977>:
>
> > @@ -0,0 +1,139 @@
> > +<?php
> > +/**
> > + * WikiaSQLIterator
> > + */
> > +
> > +class WikiaSQLIterator {
>
> Could be great if this class implemented an Iterator interface so we could
> supply it to a foreach loop instead of while. How does it sound?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/app/pull/6208/files#r24286977>.
>



-- 
Garth Webb

Engineering Manager






*Videogames  /  Entertainment  /
 Lifestyle......................................................................500
3rd Street, San Francisco, CA 94107E garth@wikia-inc.com
<garth@wikia-inc.com>C 415-652-7688WEB WIKIA <http://wikia.com/> | FACEBOOK
<http://www.facebook.com/wikia> | TWITTER <http://twitter.com/wikia>*